package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "npm-blue-tape",
+  "description": "The type definition for https://github.com/spion/blue-tape",
+  "version": "0.0.1",
+  "author": "blakeembrey",
```

Comment:
Can we kill the irrelevant fields and keep it minimal? Since it'll never be published, it'll be easier to maintain. We should also specify `"private": true` to block accidental publishing.